### ISSUE TITLE
503 error with MUC +openfire+pidgin

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#140]
```
What steps will reproduce the problem?
1.setup server with openfire +muc
2.tell gtalksms to send to both for sms
3.recieve a sms

What is the expected output?
sms to be shown in main window aswell as a chat invite to MUC

What do you see instead?
Incoming SMS: Unable to send conference room configuration form. Ex: 
forbidden(403)
and sms shown in main window

Additional informations:
GTalkSMS version = 2.1.3
Android OS version = 2.2
Phone model = I5500




```

Original issue reported on code.google.com by `cybercorey` on 10 Apr 2011 at 11:03

Attachments:
- [log.txt](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-140/comment-0/log.txt)

**Comment by GoogleCodeExporter** [comment]
```
Did you configure the conference server? Connection Settings -> Advanced 
Settings -> MUC Server

Should be something like "conference.<your_openfire.tld>".
```

Original comment by `fschm...@gmail.com` on 10 Apr 2011 at 1:15

**Comment by GoogleCodeExporter** [comment]
```
Yes, I did, it creates the room fine, its hidden but i can see it in the 
openfire admin
```

Original comment by `cybercorey` on 10 Apr 2011 at 9:11

**Comment by GoogleCodeExporter** [comment]
```
Closing this old issue, as it seems to be a problem on the server side 
configuration. Corey: Feel free to reopen or post a help request message on the 
mailing list.
```

Original comment by `fschm...@gmail.com` on 23 Jun 2011 at 9:24
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
A whole year later, new hardware, updated software and i still get this same 
damned problem.
Im guessing MUC isnt setup to work with Openfire.
Did some googling it apparntly its the way the create muc method is used.

phone: Sending SMS to Corey Davis (+64273433416): "2"
phone: SMS "2" sent to Corey Davis
phone: SMS from Corey Davis: 2
phone: ACTION_SMS_RECEIVED - Error writing to MUC: forbidden(403)
SMS from Corey Davis: 2
phone: Incoming SMS: Unable to send conference room configuration form. Ex: 
forbidden(403)

When i try enter the room from name of it via my client it gives me the room is 
locked error.
```

Original comment by `cybercorey` on 30 Jun 2012 at 4:22

**Comment by GoogleCodeExporter** [comment]
```
Here is the room that gets created by gtalksms.
```

Original comment by `cybercorey` on 30 Jun 2012 at 4:24

Attachments:
- [Capture.PNG](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-140/comment-5/Capture.PNG)

**Comment by GoogleCodeExporter** [comment]
```
This bug is probably fixed with changeset dacd0bbf2678 where room name has been 
normalized

String normalizedName = name.replaceAll(" ", "_").replaceAll("[\\W]|µ", "");
normalizedName = Normalizer.normalize(normalizedName, 
Normalizer.Form.NFD).replaceAll("[^\\p{ASCII}]", "");

To check with the next release
```

Original comment by `Florent....@gmail.com` on 30 Jun 2012 at 4:55

**Comment by GoogleCodeExporter** [comment]
```
Mean! I'd love to get this working locally =]
```

Original comment by `cybercorey` on 30 Jun 2012 at 8:51

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 2d8b6012bf39.
```

Original comment by `yako...@gmail.com` on 21 Feb 2014 at 11:03
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 4.4.5
```

Original comment by `Florent....@gmail.com` on 15 Aug 2014 at 7:18
- Changed state: **FixedInBeta**

**Comment by GoogleCodeExporter** [comment]
```
Released on Play Market as 5.0
```

Original comment by `Florent....@gmail.com` on 25 Oct 2014 at 6:23
- Changed state: **Deployed**
